lib/IDE/Utils.cpp

```diff
@@ -853,6 +853,10 @@ bool swift::ide::isBeingCalled(ArrayRef<Expr *> ExprStack) {
       if (CRCE->getBase() == Target)
         return true;
     }
+    if (auto *CRCE = dyn_cast<ConstructorRefCallExpr>(AE->getFn()))
+      if (auto *MCE = dyn_cast<MetatypeConversionExpr>(CRCE->getBase()))
+        if (MCE->getSubExpr() == Target)
+          return true;
```

Comment:
the problem is the slight difference in casing `AE->getFn` to `ConstructorRefCallExpr` instead of just `AE`. Good call on using `ImplicitConversionExpr` instead

Reply:
Ah yeah, presumably the issue is the `isImplicit` check above? I think both cases should be able to use `AE->getFn()`, as a `ConstructorRefCallExpr` can only be used as the function of another call (we build an autoclosure for the unapplied case).

Reply:
nice, done